docs/reference/config.rst

```diff
@@ -1,39 +1,37 @@
 Configuration
 =============
 
-Beets has an extensive configuration system that lets you customize nearly
-every aspect of its operation. To configure beets, you create a file called
-``config.yaml``. The location of the file depend on your platform (type ``beet
-config -p`` to see the path on your system):
-
-* On Unix-like OSes, write ``~/.config/beets/config.yaml``.
-* On Windows, use ``%APPDATA%\beets\config.yaml``. This is usually in a
-  directory like ``C:\Users\You\AppData\Roaming``.
-* On OS X, you can use either the Unix location or ``~/Library/Application
+Beets has an extensive configuration system which lets you customize nearly
+every aspect of beets. To configure beets, a configuration file called
+``config.yaml`` must be created. The location of ``config.yaml`` depends on your operating system (OS).
+
+* On a Unix OS, use ``~/.config/beets/config.yaml``.
+* On Windows, use ``%APPDATA%\beets\config.yaml``. ``%APPDATA%`` is usually an environment variable
+  which stores a directory like ``C:\Users\[Your Windows' username]\AppData\Roaming``.
+* On OS X, you can use either the Unix OS's location or ``~/Library/Application
   Support/beets/config.yaml``.
 
-You can launch your text editor to create or update your configuration by
-typing ``beet config -e``. (See the :ref:`config-cmd` command for details.) It
-is also possible to customize the location of the configuration file and even
-use multiple layers of configuration. See `Configuration Location`_, below.
+You can launch your text editor to create or update your ``config.yaml`` by
+executing ``beet config -e``. (See the :ref:`config-cmd` command for details.) The location of ``config.yaml`` 
+can be customized as described in `Configuration Location`_. Using beets, you can confirm 
+the ``config.yaml`` file's location by executing ``beet config -p``.
 
-The config file uses `YAML`_ syntax. You can use the full power of YAML, but
-most configuration options are simple key/value pairs. This means your config
-file will look like this::
+``config.yaml`` uses `YAML`_ syntax. Most of beets' configuration options are 
+key/value pairs which look like these examples::
 
     option: value
     another_option: foo
     bigger_option:
         key: value
         foo: bar
 
-In YAML, you will need to use spaces (not tabs!) to indent some lines. If you
-have questions about more sophisticated syntax, take a look at the `YAML`_
+In YAML, you will need to use spaces (not tabs!) to indent lines. If you
+have questions about YAML's syntax, you can review the `YAML`_
 documentation.
 
 .. _YAML: http://yaml.org/
 
-The rest of this page enumerates the dizzying litany of configuration options
+The remainder of this document describes the numerous configuration options that are
```

Comment:
Here’s another case where I prefer a friendlier, more whimsical diction to the technically more accurate version. While “rest” and “remainder” are synonyms, the former is less formal. Same with “page” vs. “document.” And finally, I like the flair of “dizzying litany” over the clinical “numerous”—beets really does have an overwhelming number of configuration options, and I like to poke fun at how nerdy and detailed this can seem at times.

Reply:
I see, when I first read this phrase I did not know what "litany" meant and thought that it's not a commonly used term. 

When I looked up "litany" on Google, the first definition that popped up was 
> a series of petitions for use in church services or processions, usually recited by the clergy and responded to in a recurring formula by the people.

Is there an alternative that could be used other than "numerous"?

Reply:
How about [definition #2 from Wiktionary](https://en.wiktionary.org/wiki/litany): “a prolonged or tedious list”? :smiley:

Reply:
Interesting! But I still don't think that "litany" is the right word to use. "tedious" gives the connotation that a list is too long and boring. I still don't think that "litany" is the right word to use. "selection" is more direct.

Reply:
It is direct, but I really do mean to use self-deprecating humor here—so in my mind, a “tedious” connotation, or at least “prolonged” (implying “too long”), is perfect!

Reply:
I see, I'll change it back to "dizzying litany" to give the document the voice you'd like it to have